lib/data/process-test-directory.js

```diff
@@ -528,6 +528,7 @@ const processTestDirectory = async ({ directory, args = {} }) => {
       setup_script_description: getSetupScriptDescription(test.setupScript.scriptDescription),
       setupTestPage: test.setupScript.script,
       specific_user_instruction: test.instructions,
+      assertionResponseQuestion: supportJson.testPlanStrings.assertionResponseQuestion,
```

Comment:
You should also include the same in `process-test-directory-v1`, since file also has references to `support.json`. And we'd expect to see the same string in the v1 output if `support.json` changes.